Missing required parameter for [Route: socialite.redirect] after Laravel 12.x changes

### The problem

The latest 
Running composer update socialiteproviders/saml2 socialiteproviders/microsoft
Is causing login prompt errors which are unable to pass. I'm assuming its related to Laravel environment variables?

[2025-04-29T09:42:58][ERROR] Missing required parameter for [Route: socialite.redirect] [URI: auth/{provider}/redirect] [Missing parameter: provider]. {"view":{"view":"/opt/librenms/resources/views/auth/login-form.blade.php","data":[]},"exception":"[object] (Spatie\\LaravelIgnition\\Exceptions\\ViewException(code: 0): Missing required parameter for [Route: socialite.redirect] [URI: auth/{provider}/redirect] [Missing parameter: provider]. at /opt/librenms/vendor/laravel/framework/src/Illuminate/Routing/Exceptions/UrlGenerationException.php:35)
[previous exception] [object] (Illuminate\\Routing\\Exceptions\\UrlGenerationException(code: 0): Missing required parameter for [Route: socialite.redirect] [URI: auth/{provider}/redirect] [Missing parameter: provider]. at /opt/librenms/vendor/laravel/framework/src/Illuminate/Routing/Exceptions/UrlGenerationException.php:35)"}

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 25.4.0-53-ge24d18420 (2025-04-29T04:09:48-05:00)
DB Schema | 2025_04_15_122034_laravel_11_fix_types (332)
PHP       | 8.3.6
Python    | 3.12.3
Database  | MariaDB 10.11.11-MariaDB-0ubuntu0.24.04.2
RRDTool   | 1.7.2
SNMP      | 5.9.4.pre2
===========================================

[OK]    Composer Version: 2.8.8
[OK]    Dependencies up-to-date.
[OK]    Database Connected
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQL and PHP time match
[FAIL]  Poller is not running.  No poller has run within the last 300 seconds.
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[FAIL]  No active python wrapper pollers found
[OK]    Redis is unavailable
[OK]    rrdtool version ok
[OK]    Connected to rrdcached
```

### What was the last working version of LibreNMS?

25.4.0-53-ge24d18420

### Anything in the logs that might be useful for us?

```txt
Poller stopped manually.
```

--- a/database/migrations/2025_05_02_133959_filter_empty_socialite_configs.php
+++ b/database/migrations/2025_05_02_133959_filter_empty_socialite_configs.php
@@ -1,0 +1,32 @@
+<?php
+
+use App\Facades\LibrenmsConfig;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        $configs = LibrenmsConfig::get('auth.socialite.configs', []);
+        $configs = array_filter($configs, function ($key) {
+            if (is_string($key) && strlen(trim($key)) == 0) {
+                return false;
+            }
+
+            return true;
+        }, ARRAY_FILTER_USE_KEY);
+
+        LibrenmsConfig::persist('auth.socialite.configs', $configs);
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        //
+    }
+};

